### PR TITLE
small documentation fixes

### PR DESCRIPTION
**Pull request by llelf**
**Kind:** documentation.

### CODE REVIEW

File: doc/sphinx/biblio.bib [hunk 1]
```diff
@@ -192,7 +192,7 @@ @InProceedings{CSwcu
 
 @InProceedings{Del00,
   author      = {Delahaye, D.},
-  title       = {A {T}actic {L}anguage for the {S}ystem {{\sf Coq}}},
```
Comment (cpitclaudel) [inline doc/sphinx/biblio.bib]: Maybe keep the `{{}}` for capitalization?

File: doc/sphinx/proof-engine/ssreflect-proof-language.rst [hunk 2]
```diff
@@ -1513,7 +1513,7 @@ In a goal like the following::
   =============
   m < 5 + n
 
-The tactic ``abstract: abs n`` first generalizes the goal with respect ton
+The tactic :g:`abstract: abs n` first generalizes the goal with respect to :math:`n`
```
Comment (Zimmi48) [inline doc/sphinx/proof-engine/ssreflect-proof-language.rst]: Despite this being done somewhere else, I'd advise against using the `:math:` scope in such case. `:g:` is fine.

**Review by Zimmi48 — APPROVED** [review]
Thanks! If you get the chance before I merge this, please squash the commits together.

**Comment by llelf** [comment]
@Zimmi48 done

**Pull request merged**